chapters/tutorials/tutorial_01.rst

```diff
@@ -0,0 +1,145 @@
+==================================
+Tutorial 1
+==================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+What is Jina
+----------------------------------
+
+Tutorial
+----------------------------------
+
+At the end of this tutorial, you will have your search engine using text. You will use text as an input and get a matching text for that.
+For this example, we will use the `Kaggle Wikipedia Corpus <https://www.kaggle.com/mikeortman/wikipedia-sentences>`_
+You will understand how every part of this example works and how you could create new apps with different datasets on your own.
+
+Set-up & overview
+----------------------------------
+
+The first part is to install Jina.
+
+  .. code-block:: python
+
+    pip install jina
+
+And we will also need the following dependencies:
+
+  .. code-block:: python
+
+    pip install click==7.1.2
+    pip install transformers==4.1.1
+    pip install torch==1.7.1
+
+
+Once you have Jina installed let's take a broad overview of what we should do:
+
+.. image:: res/flow.png
```

Comment:
Great image! But it's very hard to read the small text? Could we also add some colour 🌈 ?

Reply:
Im not sure about this image tbh, should we have something more jina designed? I used it as a base, I would like something like that, just not sure about the design

Reply:
We could ask the design team?

Reply:
You should reach out to Kelly/ TK directly via #team-creative-design